aiocron/__init__.py

```diff
@@ -85,10 +86,18 @@ def call_next(self):
 
     def call_func(self, *args, **kwargs):
         """Called. Take care of exceptions using gather"""
-        asyncio.gather(
-            self.cron(*args, **kwargs),
-            loop=self.loop, return_exceptions=True
-            ).add_done_callback(self.set_result)
+        """Check the version of python installed"""
+        if ((int(sys.version_info[0]) == 3 and int(sys.version_info[1]) >= 10)
```

Comment:
You can forget about python2. A simple test like this should be enough: `sys.version_info[0:2] >= (3, 10)`